<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainIhm</class>
 <widget class="QMainWindow" name="MainIhm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1885</width>
    <height>1340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Station au sol de contrôle du drone</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1851</width>
      <height>1211</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabAvant">
     <attribute name="title">
      <string>Avant mission</string>
     </attribute>
     <widget class="QGroupBox" name="gbPM">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>1801</width>
        <height>1111</height>
       </rect>
      </property>
      <property name="title">
       <string>Paramètres de la mission</string>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>240</y>
         <width>341</width>
         <height>39</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom de la mission : </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leNomMission">
       <property name="geometry">
        <rect>
         <x>700</x>
         <y>240</y>
         <width>421</width>
         <height>47</height>
        </rect>
       </property>
       <property name="text">
        <string>Apollo16</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pbTransfertDrone">
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>1020</y>
         <width>961</width>
         <height>48</height>
        </rect>
       </property>
       <property name="text">
        <string>Transférer paramètres vers le drone et activer mission</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="gbTD">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>700</y>
         <width>731</width>
         <height>371</height>
        </rect>
       </property>
       <property name="title">
        <string>Transfert distant</string>
       </property>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>70</y>
          <width>211</width>
          <height>39</height>
         </rect>
        </property>
        <property name="text">
         <string>Adresse IP :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>150</y>
          <width>101</width>
          <height>39</height>
         </rect>
        </property>
        <property name="text">
         <string>Port :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="leAddrIpD">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>60</y>
          <width>321</width>
          <height>47</height>
         </rect>
        </property>
        <property name="inputMask">
         <string>999.999.999.999</string>
        </property>
        <property name="text">
         <string>192.168.2.254</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lePortD">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>140</y>
          <width>141</width>
          <height>47</height>
         </rect>
        </property>
        <property name="inputMask">
         <string>99999</string>
        </property>
        <property name="text">
         <string>22222</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="cbTransfertActive">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>240</y>
          <width>591</width>
          <height>39</height>
         </rect>
        </property>
        <property name="text">
         <string>Transfert activé durant la mission</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pbConnecterD">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>300</y>
          <width>215</width>
          <height>48</height>
         </rect>
        </property>
        <property name="text">
         <string>Connecter</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="gbII">
       <property name="geometry">
        <rect>
         <x>1050</x>
         <y>400</y>
         <width>711</width>
         <height>561</height>
        </rect>
       </property>
       <property name="title">
        <string>Incrustations initiales</string>
       </property>
       <widget class="QCheckBox" name="cbInc">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>80</y>
          <width>351</width>
          <height>39</height>
         </rect>
        </property>
        <property name="text">
         <string>Incrustation vidéo</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QListWidget" name="lwCapteurs">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>260</y>
          <width>631</width>
          <height>271</height>
         </rect>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="currentRow">
         <number>-1</number>
        </property>
       </widget>
       <widget class="QCheckBox" name="cbDateInc">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>130</y>
          <width>121</width>
          <height>39</height>
         </rect>
        </property>
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="cbHeureInc">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>180</y>
          <width>351</width>
          <height>39</height>
         </rect>
        </property>
        <property name="text">
         <string>Heure</string>
        </property>
       </widget>
      </widget>
      <widget class="QComboBox" name="cbListePortSerie">
       <property name="geometry">
        <rect>
         <x>700</x>
         <y>310</y>
         <width>681</width>
         <height>47</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>310</y>
         <width>661</width>
         <height>39</height>
        </rect>
       </property>
       <property name="text">
        <string>Choix du port série pour la liaison data :</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pbNewMission">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>311</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string>Nouvelle mission</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>400</y>
         <width>1021</width>
         <height>261</height>
        </rect>
       </property>
       <property name="title">
        <string>Equipements du drone</string>
       </property>
       <widget class="QCheckBox" name="cbMesure">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>981</width>
          <height>39</height>
         </rect>
        </property>
        <property name="text">
         <string>Emettre les mesures en temps réel vers la station au sol</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>920</x>
          <y>190</y>
          <width>61</width>
          <height>39</height>
         </rect>
        </property>
        <property name="text">
         <string>ms</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="leTimer">
        <property name="geometry">
         <rect>
          <x>760</x>
          <y>180</y>
          <width>151</width>
          <height>47</height>
         </rect>
        </property>
        <property name="inputMask">
         <string>999999</string>
        </property>
        <property name="text">
         <string>1000</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>180</y>
          <width>741</width>
          <height>39</height>
         </rect>
        </property>
        <property name="text">
         <string>Interval de raffraîchissement des mesures :</string>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="pbTestData">
       <property name="geometry">
        <rect>
         <x>1410</x>
         <y>310</y>
         <width>271</width>
         <height>48</height>
        </rect>
       </property>
       <property name="text">
        <string>Test avec EDD</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tabPendant">
     <attribute name="title">
      <string>Pendant mission</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>1030</x>
        <y>70</y>
        <width>781</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string>Gestion de la caméra</string>
      </property>
      <widget class="QPushButton" name="pbEmettreOrdre">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>70</y>
         <width>451</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string>Emettre ordre</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cbVal">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>180</y>
         <width>131</width>
         <height>48</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>00</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>01</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>02</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>03</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>04</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>05</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>21</width>
         <height>39</height>
        </rect>
       </property>
       <property name="text">
        <string>/</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>180</y>
         <width>51</width>
         <height>39</height>
        </rect>
       </property>
       <property name="text">
        <string>p=</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cbRep">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>180</y>
         <width>176</width>
         <height>48</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>bacpac</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>camera</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>180</y>
         <width>21</width>
         <height>39</height>
        </rect>
       </property>
       <property name="text">
        <string>/</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cbApp">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>180</y>
         <width>241</width>
         <height>48</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>PW Power</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BS BIP sound</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LL Localisation</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TI Timer</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PR Photo résolution</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FV FOV ?</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SH Capture</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>VR Vidéo résolution</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UP Orientation</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CM Mode Caméra</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PV Preview</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QTextEdit" name="teFil">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>350</y>
        <width>801</width>
        <height>661</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pbStopperMission">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>240</y>
        <width>381</width>
        <height>48</height>
       </rect>
      </property>
      <property name="text">
       <string>Stopper la mission</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbDepartAcqMes">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>501</width>
        <height>48</height>
       </rect>
      </property>
      <property name="text">
       <string>Départ acquisition mesures</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbArretAcqMes">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>501</width>
        <height>48</height>
       </rect>
      </property>
      <property name="text">
       <string>Arrêt acquisition mesures</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="gbII_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>360</y>
        <width>711</width>
        <height>561</height>
       </rect>
      </property>
      <property name="title">
       <string>Incrustations</string>
      </property>
      <widget class="QCheckBox" name="cbInc2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>80</y>
         <width>351</width>
         <height>39</height>
        </rect>
       </property>
       <property name="text">
        <string>Incrustation vidéo</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QListWidget" name="lwCapteurs2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>260</y>
         <width>631</width>
         <height>271</height>
        </rect>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="currentRow">
        <number>-1</number>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbDateInc2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>130</y>
         <width>121</width>
         <height>39</height>
        </rect>
       </property>
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbHeureInc2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>180</y>
         <width>351</width>
         <height>39</height>
        </rect>
       </property>
       <property name="text">
        <string>Heure</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tabApres">
     <attribute name="title">
      <string>Après mission</string>
     </attribute>
     <widget class="QGroupBox" name="gbReseauFilaireDrone">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>881</width>
        <height>311</height>
       </rect>
      </property>
      <property name="title">
       <string>Connexion par le réseau filaire au drone</string>
      </property>
      <widget class="QPushButton" name="pbConnecterDrone">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>230</y>
         <width>731</width>
         <height>48</height>
        </rect>
       </property>
       <property name="text">
        <string>Se connecter au drone par la prise réseau</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>70</y>
         <width>321</width>
         <height>47</height>
        </rect>
       </property>
       <property name="inputMask">
        <string>999.999.999.999</string>
       </property>
       <property name="text">
        <string>192.168.2.254</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>160</y>
         <width>101</width>
         <height>39</height>
        </rect>
       </property>
       <property name="text">
        <string>Port :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lePort">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>150</y>
         <width>141</width>
         <height>47</height>
        </rect>
       </property>
       <property name="inputMask">
        <string>99999</string>
       </property>
       <property name="text">
        <string>22222</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>80</y>
         <width>211</width>
         <height>39</height>
        </rect>
       </property>
       <property name="text">
        <string>Adresse IP :</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pbTransfertDuDrone">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>380</y>
        <width>511</width>
        <height>48</height>
       </rect>
      </property>
      <property name="text">
       <string>Transférer le fichier mesure</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="teMesure">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>450</y>
        <width>1791</width>
        <height>561</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pbTransfertDistant">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>380</y>
        <width>741</width>
        <height>48</height>
       </rect>
      </property>
      <property name="text">
       <string>Transférer les mesures vers le site distant</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbNewMission_2">
      <property name="geometry">
       <rect>
        <x>1468</x>
        <y>30</y>
        <width>341</width>
        <height>48</height>
       </rect>
      </property>
      <property name="text">
       <string>Nouvelle mission</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabHistorique">
     <attribute name="title">
      <string>Historique</string>
     </attribute>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>170</y>
        <width>711</width>
        <height>39</height>
       </rect>
      </property>
      <property name="text">
       <string>Nécessiterait une base de données...</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1885</width>
     <height>44</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Fichier">
    <property name="title">
     <string>&amp;Fichier</string>
    </property>
    <addaction name="action_Quitter"/>
   </widget>
   <widget class="QMenu" name="menu_Options">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <addaction name="action_Maintenance"/>
   </widget>
   <widget class="QMenu" name="menu_Aide">
    <property name="title">
     <string>&amp;Aide</string>
    </property>
    <addaction name="actionA_propos"/>
   </widget>
   <addaction name="menu_Fichier"/>
   <addaction name="menu_Options"/>
   <addaction name="menu_Aide"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Quitter">
   <property name="text">
    <string>&amp;Quitter</string>
   </property>
  </action>
  <action name="action_Maintenance">
   <property name="text">
    <string>&amp;Maintenance</string>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="text">
    <string>A &amp;propos</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
